openshift_resources/resource.py

```diff
@@ -6,7 +6,12 @@
 from resources.io.k8s.apimachinery.pkg.apis.meta.v1 import ObjectMeta
 
 
-class BaseResource(BaseModel, metaclass=ABCMeta):
+class NewBaseModel(BaseModel):
```

Comment:
is this correct?

Reply:
yes, by default BaseModel doesn't know how to deserialize Eums, so we need to overide 'use_enum_values' it in it's inner class Config to resolve it; by default it's false.